bin/build-plugin-zip.sh

```diff
@@ -121,6 +121,7 @@ zip -r gutenberg.zip \
 	gutenberg.php \
 	lib/*.php \
 	core-blocks/*/*.php \
+	packages/core-blocks/src/*/*.php \
```

Comment:
I think we agreed to rename this stuff to `blocks-library` or something along those lines. /cc @mtias